src/context/HicetnuncContext.js

```diff
@@ -162,53 +165,39 @@ class HicetnuncContextProviderClass extends Component {
 
       },
 
-      // marketplace v2
-
-      collectv2: async (swap_id, xtz_amount) => {
-        return await Tezos.wallet
-          .at(this.state.v2)
-          .then((c) =>
-            c.methods
-              .collect(parseFloat(swap_id))
-              .send({
-                amount: parseFloat(xtz_amount),
-                mutez: true,
-                storageLimit: 310,
-              })
-          )
-      },
-
-      swapv2: async (from, royalties, xtz_per_objkt, objkt_id, creator, objkt_amount) => {
+      swapTeia: async (from, royalties, xtz_per_objkt, objkt_id, creator, objkt_amount) => {
         // If using proxy: both calls are made through this.state.proxyAddress:
         const objktsAddress = this.state.proxyAddress || this.state.objkts;
-        const marketplaceAddress = this.state.proxyAddress || this.state.v2;
+        const marketplaceAddress = this.state.proxyAddress || MARKETPLACE_CONTRACT_TEIA;
```

Comment:
This would not work in the case when proxy is active, because later in `marketplace.methods.swap` proxy will await parameters for v2 swap and transaction wouldn't be accepted. This is the place where we need to implement different logic for proxy swaps (with loading lambda, packing params and making call to `execute`)

Reply:
i changed the code so that it uses the v2 contract in this case for now. when the `swapCollab` and `reswapCollab` methods are ready, i'll call them here.